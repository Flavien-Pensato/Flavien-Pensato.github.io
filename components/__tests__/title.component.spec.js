import React from 'react'
import { shallow } from 'enzyme'
import serializer from 'jest-emotion'
import { Title } from '../title.component'

expect.addSnapshotSerializer(serializer)

jest.unmock('../title.component.js')

describe('<Title />', () => {
  const getShallow = () => shallow(<Title />)

  it('should match snapshot', () => {
    expect(getShallow()).toMatchSnapshot()
  })
})
