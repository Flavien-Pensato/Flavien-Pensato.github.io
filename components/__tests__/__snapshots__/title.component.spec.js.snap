// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Title /> should match snapshot 1`] = `
<Styled(header)>
  <Styled(h1)>
    <Styled(i)
      className="snes-jp-logo brand"
    />
    Hello,
    <br />
    I'm Flavien!
    <br />
  </Styled(h1)>
  <Styled(p)>
    An enthusiastic Frontend developper
  </Styled(p)>
</Styled(header)>
`;
